admin_manual/configuration_server/performance_tuning/webserver_tips.rst

```diff
@@ -12,6 +12,15 @@ on/off``. If it is on, then add this line to ``php.ini`` to turn it off::
 
  safe_mode = Off
 
+Raise max_file_uploads of PHP
+-----------------------------
+
+The PHP setting ``max_file_uploads`` within the ``php.ini`` defaults to ``20``
+on most environments which allows that number of simultaneously uploads.
```

Comment:
Should - "simultaneous uploads" is the correct adjective+ noun.
If you write it "allows that number of uploads to happen simultaneously" - "simultaneously" is the adverb describing the manner in which "to happen" happens.